api_tests/lib_sdk/utils.ts

```diff
@@ -83,3 +84,11 @@ export async function createDefaultSwapRequest(counterParty: Actor) {
     };
     return swapRequest;
 }
+
+export async function fileExists(filepath: string) {
```

Comment:
Would be good if the fn name reflects the waiting until the file actually exists.

Reply:
Haha @tcharding

Reply:
Yep, that one was me :)

Reply:
Suggestions: `untilFileExists`, reads nicely with `await`:

```js
await untilFileExists(...)
```

Reply:
Changed to `waitUntilFileExists` as @D4nte  originally had it :)